mandatory/2-function-creation.js

```diff
@@ -25,7 +41,11 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  return arrayOfStrings.map((array) =>
```

Comment:
We'll cover the Array map() method more in Week 4. The logic here looks correct to me. Something to mention here is that the parameter of the function being passed to the map method is called `array`, but it's not really an array, it's actually a string in this case.